x/upgrade/keeper/keeper.go

```diff
@@ -74,85 +72,47 @@ func (k Keeper) ScheduleUpgrade(ctx sdk.Context, plan types.Plan) error {
 
 	store := ctx.KVStore(k.storeKey)
 
-	// clear any old IBC state stored by previous plan
-	oldPlan, exists := k.GetUpgradePlan(ctx)
```

Comment:
I believe this should stay to separate concerns. We're still clearing ibc state in `ApplyUpgrade` anyway.